### PR TITLE
Ported the app from ios to Android

### PR DESCRIPTION
**Pull request by AliBaWazir**
Android version of the basic app is now working

### CODE REVIEW

File: index.android.js [hunk 1]
```diff
@@ -9,45 +9,186 @@ import {
...
+  subTitle:{
+    fontFamily:'HelveticaNeue-Medium',
+    fontSize:24,
+    color:'white',
+    marginBottom:-444
+  }
+})
 
-AppRegistry.registerComponent('SimpleWeatherApp', () => SimpleWeatherApp);
+AppRegistry.registerComponent('SimpleWeatherApp', () => App);
```
Comment (buamod) [inline index.android.js:194]: Why did you change the name of the App to App ? it should be SimpleWeatherApp

**Review by buamod — COMMENTED** [review]
check my comment

File: index.android.js (hunk 1 above)
Comment (AliBaWazir) [reply index.android.js:194]: The app name is determined by what's between the '' marks. in this case it is 'SimpleWeatherApp'. The word App in this case is the class name.

Comment (buamod) [reply index.android.js:194]: 👍
